app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,11 @@
+class DashboardController < ApplicationController
+
+  def index
```

Comment:
i like that you're grabbing this data in your controller and passing it in via instance variables